src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -164,16 +166,10 @@ const HotspotDetails = ({
   }, [address, dispatch, hotspot, listIndex, prevListIndex, timelineValue])
 
   useEffect(() => {
-    if (!address) return
+    if (!address || !hotspot || !hotspotSyncBuffer) return
 
-    dispatch(
-      fetchSyncStatus({
-        address,
-        statusTime: hotspot?.status?.timestamp,
-        blockHeight,
-      }),
-    )
-  }, [address, blockHeight, dispatch, hotspot?.status?.timestamp])
+    dispatch(fetchSyncStatus({ hotspot, hotspotSyncBuffer }))
```

Comment:
feels a little odd for a view to know about sync buffer?

Reply:
I agree. I guess I could pull it inside the slice from state. That's probably better.